docs/project/2024/alecsandra_elena.iordache/index.md

```diff
@@ -0,0 +1,92 @@
+# Saving the Flower
+
+Rust-powered environmetal monitoring the umidity percentage of the plant
+
+:::info 
+
+**Author**: Iordache Alecsandra-Elena \
+**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-alecsandraiordache
+
+:::
+
+## Description
+
+My project focuses on innovative humidity sensing using the Raspberry Pi Pico microcontroller. I aim to address the critical need for efficient humidity control in various environments. Leveraging advanced sensing technology and the versatile Raspberry Pi Pico, we're developing a solution for real-time monitoring and regulation of humidity levels.
+
+## Motivation
+
+Think about it – every drop counts, right? By setting up this cool project, we're actually doing our part for the planet. Water scarcity is a big deal nowadays, and we're tackling it head-on by making sure we're not wasting a single drop. With my system in place, we're not just watering plants efficiently; we're also being kinder to Mother Earth. It's like showing her some love and saying, "Hey, we care about you!" Plus, when others see what we're doing, maybe they'll get inspired to jump on the eco-friendly bandwagon too. So, it's not just about plants – it's about making a difference, one humidity sensor at a time!
+
+## Architecture 
+![A prototype of architecture](architecture.jpg)
```

Comment:
Change the file's extension so that it matches with the picture saved (architecture.png). Otherwise the build to github-pages check will fail.

Reply:
Test deployment test failed. Please fix this when you can.